backend/molgenis-emx2-io/src/test/java/org/molgenis/emx2/io/TestLegacyImport.java

```diff
@@ -76,23 +76,23 @@ private void executeTest(TableStore store, Schema schema) {
         schema
             .getTable("biobanks")
             .select(
-                s("name"),
-                s("contact_person", s("full_name")),
-                s("principal_investigators", s("full_name")),
-                s("juristic_person", s("name")))
+                s("Name"),
```

Comment:
This is important: when importing emx1 we now use 'label' instead of 'id' as the name.